.github/workflows/go.yml

```diff
@@ -50,10 +50,12 @@ jobs:
         path: ${{ steps.go-cache-paths.outputs.go-mod }}
         key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
 
-
     - name: Get dependencies
       run: make deps
 
+    - name: Run govulncheck
+      run: go install golang.org/x/vuln/cmd/govulncheck@latest; govulncheck ./...
```

Comment:
Feels cleaner as different lines, or different steps. But I'll take it. 

Side note, do you think in here, or in a new workflow parallel?

Reply:
I _think_ here is better, it will fail fast, but it means if a new vuln is reported, it will stop all CI ... do we want that? We'll be forced to remediate things immediately. I think we do, but would like to hear any differing opinion.

Reply:
I don't have strong feelings.  

Here should have good coverage in the different platforms. But it fails fast. 

Seems fine to call this good enough until it's in the main linter.